src/shared/sam/debugger/samDebugSession.ts

```diff
@@ -126,46 +130,53 @@ export class SamDebugSession extends DebugSession {
         this.sendEvent(new InitializedEvent())
     }
 
+    protected async launchRequest(response: DebugProtocol.LaunchResponse, launchConfig: SamLaunchRequestArgs) {
```

Comment:
Just want to make sure I understand, this and attachRequest aren't hooked up to anything right now?

Reply:
attachRequest is called by vscode implicitly